django/contrib/admin/static/admin/js/unusable_password_field.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+// Fallback JS for browsers which do not support :has selector used in
+// admin/css/unusable_password_fields.css
+// Remove file once all supported browsers support :has selector
+try {
+    // If browser does not support :has selector this will raise an error
+    document.querySelector("form:has(input)");
+} catch (error) {
```

Comment:
But I'll defer to @nessita's decision on this 😄

Reply:
I disagree, what you're proposing is a really aggressive policy :fire: . We cannot drop support for 1 months old Firefox releases just because the latest one supports `has()`. I (for example) have 119 locally :older_adult:. We need to remember that admin is for app users and not all of them bump Firefox daily.

Reply:
I was looking at this with a lens of what the state of things will look in 8 months time rather than right now. In that regard we'll [be expecting](https://whattrainisitnow.com/calendar/) v129 to be the latest version[^1]. _(Also v128 will be the next ESR and, yes, there'll be one more release of v115 ESR to come in September before it hits end-of-life in October.)_

Our documentation states "recent modern versions" and I get that this is quite nebulous, but I would hardly call what I proposed "really aggressive" in light of that. I will concede though that, in this case, it's likely to cause more than "minor stylistic differences" to the feature if the browser doesn't have the required support.

Regardless, I was just making a case that including this seemed pointless and, if that's rejected, it's fine ❤️

[^1]: I certainly hope you've upgraded to something more recent than v119 by then for your own security 😛

Reply:
I have thoroughly tested the three paths:
1. Browser with `has` support
2. No `has` support and JS enabled
3. No `has` support and JS disabled

User experience is quite decent in all cases (after I made some tweaks to the forms and the view flows). I think we can safely land as is and cleanup in a future iteration. Thank you!